### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by dgarcia360**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,54 +1,13 @@
-# catbuffer
+# catbuffer-generators
 
 [![Build Status](https://api.travis-ci.org/nemtech/catbuffer.svg?branch=master)](https://travis-ci.org/nemtech/catbuffer-generators)
```
Comment (Jaguar0625) [inline README.md]: does catbuffer.svg need to be updated too ?

File: README.md [hunk 2]
```diff
@@ -1,54 +1,13 @@
...
-|----------------------|---------------------------------------------------------|---------------|
-| -s, --schema TEXT    | input CATS file                                         |               |
-| -o, --output TEXT    | output directory                                        | _generated    |
-| -i, --include TEXT   | schema root directory                                   | ./schemas     |
-| -g, --generator TEXT | generator to use to produce output files                |               |
-| -c, --copyright TEXT | file containing copyright data to use with output files | ../HEADER.inc |
-
-
-## Examples
+## Supported programming languages
```
Comment (Jaguar0625) [inline README.md]: remove trailing tab

**Comment by Jaguar0625** [comment]
i think this was already merged @dgarcia360 ?

**Comment by dgarcia360** [comment]
@dgarcia360 We merged the catbuffer repo README but not this one (generadors).

**Comment by Jaguar0625** [comment]
f7e8fab2349b78114730e652feb3aac740f55e9c

**Pull request closed without merging**